spec.bs

```diff
@@ -499,6 +519,33 @@ to conduct an auction to select an advertisement to display to the user, they ca
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
 
+<h3 id="createauctionnonce">createAuctionNonce()</h3>
+
+{{Window/navigator}}.{{Navigator/createAuctionNonce()}} creates an auction nonce, a one-time ID

```

Comment:
How can we communicate that, for multi-seller auctions, this is called and produces a one-time ID uniquely associated with each distinct component seller and their auction config? Perhaps we can say something like, a one-time ID uniquely associated with each distinct `AuctionConfig`? (Does `AuctionConfig` here mean the top-level thing or the thing that specifies `interestGroupBuyers`?)

Reply:
I've thought about that, but know what's a good way for that. I'll keep this open and ask our spec mentor's opinion when I ask them to review.

Reply:
@domfarolino Any suggestion on this? Thanks!

Reply:
Am I correct in thinking you're just trying to describe something that already happens/falls-out from the actual algorithms? In that case, I guess I'd recommend just adding a small [Non-normative paragraph or note](https://wicg.github.io/fenced-frame/#:~:text=2.3.1.%20Introduction-,This%20section%20is%20non%2Dnormative,-.) describing exactly what the algorithms do in this tricky case that's not super obvious from the surface. How does that sound?

Reply:
Added a sentence "For
multi-seller auctions, this ID is uniquely associated with all {{AuctionAdConfig/componentAuctions}}". Let me know if there's any suggestion.